tests/vitest-vscode.ts

```diff
@@ -0,0 +1,18 @@
+import { runInRepo } from '../utils'
+import { RunOptions } from '../types'
+
+// npx tsx ecosystem-ci.ts vitest-vscode --release 1.3.1
+
+export async function test(options: RunOptions) {
+	await runInRepo({
+		...options,
+		build: 'compile',
+		repo: 'vitest-dev/vscode',
+		test: 'test',
+
```

Comment:
The https://github.com/vitest-dev/vscode/pull/276 seems to add `playwright`. For this we'll need to install its binaries in here. Similar as `vitest-tests/browser-simple` does:

https://github.com/vitest-dev/vitest-ecosystem-ci/blob/7a1d904ee48bb524cc4d6ebe811459c7f8d414d5/tests/vitest-browser-simple.ts#L9

Reply:
Currently both `test` and `test-e2e` automatically downloads Vscode binary using `@vscode/test-electron`, so this should not be necessary (and actually there's no `playwright install` command for electron).

Reply:
One thing I need to do is to wrap these commands by `xvfb-run` since there's no headless mode.

Probably this will work on Ubuntu CI, but `xvfb-run` needs to be injected conditionally, so that it can also run locally:

```ts
		test: async () => {
			await $`xvfb-run --auto-servernum --server-args='-screen 0 1024x768x24' pnpm test`;
			await $`xvfb-run --auto-servernum --server-args='-screen 0 1024x768x24' pnpm test-e2e`;
		},
```